src/Apps/Order/Components/ExpressCheckout/index.tsx

```diff
@@ -29,9 +29,16 @@ type Seller = Exclude<
 
 export const ExpressCheckout = ({ order, setShowSpinner }: Props) => {
   const orderData = useFragment(ORDER_FRAGMENT, order)
+  const [isChrome, setIsChrome] = useState<boolean | null>(null)
+
+  useEffect(() => {
+    const chrome =
+      typeof navigator !== "undefined" && /Chrome/.test(navigator.userAgent)
+    setIsChrome(chrome)
+  }, [])
+
+  if (isChrome === null) return null
```

Comment:
Oh, maybe i'm wrong. Do we need it to be momentarily null and returning nothing? Is this just to cover the server-side render?